thingsboard_gateway/storage/sqlite/storage_settings.py

```diff
@@ -12,10 +12,25 @@
 #     See the License for the specific language governing permissions and
 #     limitations under the License.
 
+from os import path
+
+
 class StorageSettings:
     def __init__(self, config):
         self.data_folder_path = config.get("data_file_path", "./")
```

Comment:
Looks like at the moment, this parameter means data_file_path, not data_folder_path anymore. Please rename it.